gateway/handlers/proxy.go

```diff
@@ -140,7 +139,7 @@ func invokeService(w http.ResponseWriter, r *http.Request, metrics metrics.Metri
 	contentType := r.Header.Get("Content-Type")
 	fmt.Printf("[%s] Forwarding request [%s] to: %s\n", stamp, contentType, url)
 
-	request, err := http.NewRequest("POST", url, bytes.NewReader(requestBody))
+	request, _ := http.NewRequest("POST", url, r.Body)
```

Comment:
Should probably not swallow err. Have you tested this change?

Reply:
the error was swallowed before my change I just made it obvious so that `go vet` stops complaining. There was no error handling, and I can not imagine a case where http.NewRequest() should fail.